### PR TITLE
Added Raises block to all the functions

### PR DESCRIPTION
**Pull request by ShravanDoda**
#325 
Fixes #

### Checklist
- [x] My branch is up-to-date with upstream/develop branch.
- [x] Everything works and tested for Python 3.5.2 and above.

### Description

I've added Raises block to docstrings of all the functions.
### Change logs

Raises blocks to docstrings

### CODE REVIEW

File: hydrus/data/exceptions.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ def get_HTTP(self) -> Tuple[int, Dict[str, str]]:
 class InstanceNotFound(Exception):
     """Error when the Instance is not found."""
 
-    def __init__(self, type_: str, id_: Union[str, None]=None) -> None:
+    def __init__(self, type_: str, id_: Union[str, None] = None) -> None:
```
[the remark this answers is not in the document]
Comment (ShravanDoda) [reply hydrus/data/exceptions.py]: Thanks. I've formatted the code. Kindly review.

**Comment by Mec-iS** [comment]
Whenever you need to rollback some code, you should use `git reset` and `git stash` to rollback to the point you need and then re-commit new code. You can save the changes you want to keep and re-add them after the reset: https://stackoverflow.com/questions/4114095/how-to-revert-a-git-repository-to-a-previous-commit

**Comment by chrizandr** [comment]
Please rebase with the new commits merged into the branch.

**Comment by ShravanDoda** [comment]
Sorry. I’m new to github. I’ll mend it as soon as I can(probably by tonight)

**Comment by chrizandr** [comment]
You need not squash all commits to one, just rebase with the new changes that have been merged into the branch.

Just do `git rebase <branchname>` and resolve conflicts if there are any.

**Pull request merged**